README.md

```diff
@@ -280,6 +280,72 @@ SLF4J Logger.
   metricRegistry.addReporter(reporter);
 ```
 
+#### Histograms
+
+In the current implementation, clients must define the distribution buckets and associate them in the reporter with the name of the metric to be histogrammed.
+
+There two types of distribution buckets available:
+- `DistributionBucket` represented by a primitive `long` array.
+- `DoubleValuedDistributionBucket` represented by a primitive `double` array
```

Comment:
Would it be more concise to call it `DoubleDistributionBucket`? Just wondering, let's chat before changing :)

Reply:
Even I was considering that too. This is the similar debate that we had while naming the `GaugeDouble`.